src/include/daos_srv/vos.h

```diff
@@ -1037,7 +1041,8 @@ vos_iterate(vos_iter_param_t *param, vos_iter_type_t type, bool recursive,
 int
 vos_obj_query_key(daos_handle_t coh, daos_unit_oid_t oid, uint32_t flags,
 		  daos_epoch_t epoch, daos_key_t *dkey, daos_key_t *akey,
-		  daos_recx_t *recx, struct dtx_handle *dth);
+		  daos_recx_t *recx, unsigned int cell_size, unsigned int stripe_size,
```

Comment:
looks better to use uint64_t for stripe_size (may also for cell_size).

Reply:
cell size use unsigned int in daos_oclass_attr.

Reply:
but stripe_size seems should be defined as uint64_t just for safe, same as other places in code, can be changed in following patch.

Reply:
this actually a bit confused here, I thought both cell_size and stripe_size should be 32 bit. I saw somewhere in cli_ec.c still use 32bit for stripe size.